doc/api/errors.md

```diff
@@ -250,10 +250,12 @@ not capture any frames.
 
 #### error.message
 
-Returns the string description of error as set by calling `new Error(message)`.
+The `error.message` property is the string description of error as set by calling
+`new Error(message)`.
 The `message` passed to the constructor will also appear in the first line of
```

Comment:
There should be either a paragraph break, or the lines should be wrapped together. It's not immediately clear what this is trying to do.